config/registry/config.yml

```diff
@@ -0,0 +1,23 @@
+# https://docs.docker.com/registry/configuration/
+
+version: 0.1
+log:
+  level: debug # info, warn, error
```

Comment:
Do we want to leave this as debug now that it's ready or should we change it to one of the other levels?

Reply:
Let's change back to `info` and also get rid of `--v=2 --alsologtostderr` in `command: --v=2 --alsologtostderr /config/auth_config.yml`, which we added to see errors.